src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_horizontal_layout.html.twig

```diff
@@ -25,13 +25,18 @@ col-sm-2
 {# Rows #}
 
 {% block form_row -%}
-    <div class="form-group{% if (not compound or force_error|default(false)) and not valid %} has-error{% endif %}">
-        {{- form_label(form) -}}
-        <div class="{{ block('form_group_class') }}">
-            {{- form_widget(form) -}}
-            {{- form_errors(form) -}}
+    {% set displayMarkup = form.children|length == 0 or form.vars.label is not same as(false) %}
```

Comment:
Actually I misread the condition, IIUC you force the label (even `false`) to be printed when there is no child.
So I guess this should be right.